src/BlazorSignalR/BlazorSignalR.csproj

```diff
@@ -8,20 +8,21 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);IncludeP2POutput</TargetsForTfmSpecificBuildOutput>
-    <Version>0.6.0-blazor-3.0.0-preview4-19216-03</Version>
+    <Version>0.7.0-blazor-3.0.0-preview6.19307.2</Version>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.0.0-preview4-19216-03" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.Browser" Version="3.0.0-preview4-19216-03" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.0.0-preview4-19216-03" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0-preview4.19216.2" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.0.0-preview6.19307.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Browser" Version="3.0.0-preview6.19307.2" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.0.0-preview6.19307.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0-preview6.19304.6" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\BlazorSignalR.JS\BlazorSignalR.JS.csproj" />
+    <ProjectReference Include="..\BlazorSignalR.JS\BlazorSignalR.JS.csproj">
+      <PrivateAssets>All</PrivateAssets>
```

Comment:
I'm not sure about this. Without this change the BlazorSignalR package got a dependency on the BlazorSignalR.JS package, which does not exist. But there might be another way... 🤔